src/components/Navbar/Navbar.jsx

```diff
@@ -1,73 +1,94 @@
 import "../Navbar/Navbar.css";
-import React, { PureComponent } from 'react'
+import React, {PureComponent} from 'react'
 // // import { Link, NavLink } from "react-router-dom";
-import { logo as LogoImage} from "../../assets/images/Products/Products";
-import { Link, NavLink } from "react-router-dom";
-import { useAuth } from "../../context/AuthContext";
-import { useCart } from "../../context/CartContext";
+import {logo as LogoImage} from "../../assets/images/Products/Products";
+import {Link, NavLink} from "react-router-dom";
+import {useAuth} from "../../context/AuthContext";
+import {useCart} from "../../context/CartContext";
```

Comment:
There should be space between the braces. Maybe use a formatter like prettier to format your code.

Reply:
done